### DISCUSSION TITLE
Texture of Arrokoth?

### DISCUSSION THREAD

**Discussion by stephanpls** [Stellarium/stellarium#2145] [Q&A]
As many of us know, the spacecraft New Horizons had a 3500 km encounter with Arrokoth ('Ultima Thule') just three years ago on 2019-01-01. I have the data ready to replay this flyby in Stellarium, which I'll share later. 
It would be good if we can see the 'real' Arrokoth in Stellarium, so we need a texture. Is it possible to make a texture of it (because of its irregular shape that might be difficult), and if yes, who would like to do it?  I don't have experience in making textures.

<img width="160" alt="Arrokoth" src="https://user-images.githubusercontent.com/96252651/147877724-3cdf6fb1-c7e9-4c2a-ad78-e05a85a0606d.png">

**Comment by alex-w** [comment]
See the shape of Arrokoth. To see a real shape in Stellarium someone should create a 3D model of celestial body (see Phobos as example)
